.k8s/components/jobs/restore/container.ts

```diff
@@ -1,31 +1,27 @@
-import env from "@kosko/env";
-import { ok } from "assert";
-
-import { EnvVar } from "kubernetes-models/v1/EnvVar";
 import { restoreContainerJob } from "@socialgouv/kosko-charts/components/azure-storage/restore-container.job";
+import { EnvVar } from "kubernetes-models/v1/EnvVar";
+import { GITLAB_LIKE_ENVIRONMENT_SLUG } from "../../../utils/GITLAB_LIKE_ENVIRONMENT_SLUG";
 
-ok(process.env.SOURCE_CONTAINER);
-ok(process.env.DESTINATION_CONTAINER);
-ok(process.env.SOURCE_SERVER === "prod" || process.env.SOURCE_SERVER === "dev");
-ok(
-  process.env.DESTINATION_SERVER === "prod" ||
-    process.env.DESTINATION_SERVER === "dev"
-);
-
-const manifests = restoreContainerJob({
+const job = restoreContainerJob({
   env: [
     new EnvVar({
       name: "SOURCE_CONTAINER",
-      value: process.env.SOURCE_CONTAINER,
+      value: "cdtn",
     }),
     new EnvVar({
       name: "DESTINATION_CONTAINER",
-      value: process.env.DESTINATION_CONTAINER,
+      value: "cdtn-dev",
```

Comment:
we had a case where the preprod container was copied to production when "ready"

Reply:
Wat ? 
@lionelB are we still doing this preprod to prod thing ?

Reply:
from what i know restore container is used at two steps :

 - feature-branch : restore `dev/cdtn` to `dev/cdtn-dev`
 - cdtn data update : restore `dev/cdtn` to `prod/cdtn`

Reply:
je confirme